airbyte-scheduler/persistence/src/main/resources/example_config.json

```diff
@@ -5,6 +5,27 @@
   "empty_string": "",
   "null_value": null,
   "one_of": {
+    "type_key": "foo",
     "some_key": 100
+  },
+  "const_object": {
+    "sub_key": "bar",
+    "sub_array": [1, 2, 3],
+    "sub_object": {
+      "sub_sub_key": "baz"
+    }
+  },
```

Comment:
BTW a typical usage in our connectors is to use under a `oneOf` node some objects to handle additional credentials fields depending on the type of auth method. So we often include a const field as part of the nested object in order to identify which auth method is being selected.

Could we also include such a case in the tests to reflect this common usage in our connectors?

Reply:
yes ok!